src/products/components/ProductStocks/ProductStocks.tsx

```diff
@@ -355,6 +441,189 @@ const ProductStocks: React.FC<ProductStocksProps> = ({
+                    )
+                  : intl.formatMessage({
+                      defaultMessage: "Unlimited",
+                      id: "unlimitedUnitsLeft",
+                      description: "section header"
+                    })}
+              </Typography>
+            )}
+          </div>
+        </CardContent>
+      )}
+
+      {productVariantChannelListings?.length > 0 && data.isPreorder && (
+        <Table>
+          <colgroup>
+            <col className={classes.colName} />
+            <col className={classes.colSoldUnits} />
+            <col className={classes.colThreshold} />
+          </colgroup>
+          <TableHead>
+            <TableRow>
+              <TableCell className={classes.colName}>
+                <FormattedMessage
+                  defaultMessage="Channels"
+                  description="tabel column header"
+                />
+              </TableCell>
+              <TableCell className={classes.colSoldUnits}>
+                <FormattedMessage
+                  defaultMessage="Sold units"
+                  description="table column header, sold units preorder quantity"
+                  id="tableColSoldUnits"
+                />
+              </TableCell>
+              <TableCell className={classes.colThreshold}>
+                <FormattedMessage
+                  defaultMessage="Channel Threshold"
+                  description="table column header"
+                  id="tableColChannelThreshold"
+                />
+              </TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {renderCollection(productVariantChannelListings, listing => {
+              if (!listing) {
+                return;
+              }
+
+              return (
+                <TableRow key={listing.id}>
+                  <TableCell className={classes.colName}>
+                    {listing.name}
+                  </TableCell>
+                  <TableCell className={classes.colQuantity}>
+                    {listing?.unitsSold || 0}
+                  </TableCell>
+                  <TableCell className={classes.colQuantity}>
+                    <TextField
+                      disabled={disabled}
+                      fullWidth
+                      inputProps={{
+                        className: classes.input,
+                        min: 0,
+                        type: "number"
+                      }}
+                      placeholder={intl.formatMessage({
+                        defaultMessage: "Unlimited"
+                      })}
+                      onChange={e => {
+                        onVariantChannelListingChange(listing.id, {
+                          costPrice: listing.costPrice,
+                          price: listing.price as undefined,
+                          preorderThreshold: +e.target.value
+                        });
+                      }}
+                      value={listing?.preorderThreshold || ""}
```

Comment:
Is `0` a valid threshold?
```suggestion
                      value={listing?.preorderThreshold ?? ""}
```

Reply:
Yes, it should be - fixed